src/com/dotcms/api/system/event/Payload.java

```diff
@@ -18,7 +20,7 @@
 	private final Object data;
 	private final Visibility visibility;
 	private final String  visibilityId; // user id or role uid, if it is global, this is not need
-
+	private final String userId;
```

Comment:
The visibilityId no allways hold the visibilityId, it was remove because the contentlet already has the userId so it neccesary here